arch/platform/cooja/Makefile.cooja

```diff
@@ -67,7 +67,7 @@ COOJA_INTFS	= beep.c ip.c leds-arch.c moteid.c \
 COOJA_CORE = platform.c mtype.c random.c sensors.c leds.c gpio-hal-arch.c buttons.c
 
 # (COOJA_SOURCEFILES contains additional sources set from simulator)
-CONTIKI_TARGET_SOURCEFILES = \
+CONTIKI_TARGET_SOURCEFILES += \
```

Comment:
`doc/getting-started/The-Contiki-NG-build-system.md` describes this variable as:

>  It contains the list of C files (`CONTIKI_TARGET_SOURCEFILES`) that the platform adds to the Contiki-NG system.

which I interpret as "this is a system variable not meant to be modified by applications".